src/Options.js

```diff
@@ -5,6 +5,25 @@
  * `.reset` functions.
  */
 
+const Style = require("./Style");
+
+const sizeStyleMap = [ // [textsize, scriptsize, scriptscriptsize]
```

Comment:
I agree this shouldn't be block, but if we defer please add a comment in the code.